content/en/docs/contribute/localization.md

```diff
@@ -120,6 +120,58 @@ Provide guidance to localization contributors in the localized `README-**.md` fi
 
 After you create the localized README, add a link to the file from the main English `README.md`, and include contact information in English. You can provide a GitHub ID, email address, [Slack channel](https://slack.com/), or other method of contact. You must also provide a link to your localized Community Code of Conduct.
 
+### Setting up the OWNERS files
+
+To set the roles of each user contributing to the localization, create an `OWNERS` file inside the language-specific subdirectory with:
+
+- **reviewers**: A list of kubernetes teams with reviewer roles, in this case, the `sig-docs-**-reviews` team created in [Add your localization team in GitHub](#add-your-localization-team-in-github).
+- **approvers**: A list of kubernetes teams with approvers roles, in this case, the `sig-docs-**-owners` team created in [Add your localization team in GitHub](#add-your-localization-team-in-github).
+- **labels**: A list of GitHub labels to automatically apply to a PR, in this case, the language label created in [Configure the workflow](#configure-the-workflow).
+
+More information about the `OWNERS` file can be found at [community/contributors/guide/owners.md](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md).
+
+The [Spanish OWNERS file](https://github.com/kubernetes/website/blob/master/content/es/OWNERS), with language code `es`, looks like:
```

Comment:
nit: can we use git.k8s.io/website/content/es/OWNERS?

Reply:
✅ 

Is the usage of `git.k8s.io` and `go.k8s.io` referenced in any style guide?

Reply:
> Is the usage of git.k8s.io and go.k8s.io referenced in any style guide?

https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#links

Reply:
Thanks!